lib/router/router.js

```diff
@@ -219,6 +228,27 @@ function getImageStepsFromRoute(imageStepsStr) {
   });
 }
 
+// This function is used to translate an image step from gcp image service to a supported format.
+function translateGISParams(parameter) {
+  var $this = this
+  if (parameter.startsWith('s')) {
+    var paramVal = parameter.substr(1)
+    var translatedParams = 'w' + $this.options.paramValDelimiter + paramVal + $this.options.paramKeyDelimiter + 'h' + $this.options.paramValDelimiter + paramVal
+    return translatedParams
+
+  } else if (parameter.startsWith('w')) {
+    var paramVal = parameter.substr(1)
+    var translatedParams = 'w' + $this.options.paramValDelimiter + paramVal
+    return translatedParams
+  } else if (parameter.startsWith('h')) {
+    var paramVal = parameter.substr(1)
+    var translatedParams = 'h' + $this.options.paramValDelimiter + paramVal
+    return translatedParams
+  } 
+
+  throw new Error('Google Image Service parameter is invalid: ' + parameter);
```

Comment:
this function is used within a `map` so it should never throw. Instead I'd just handle an undefined return value and log accordingly.